x-pack/plugins/security_solution/cypress/e2e/detection_alerts/detection_page_filters.cy.ts

```diff
@@ -48,13 +49,14 @@ const assertFilterControlsWithFilterObject = (filterObject = DEFAULT_DETECTION_P
       expect(sub.eq(idx).text().replace(',', '')).eq(
         filter.selectedOptions && filter.selectedOptions.length > 0
           ? filter.selectedOptions.join('')
-          : 'Any'
+          : ''
       );
     });
   });
 };
 
-describe('Detections : Page Filters', () => {
+// Skipped because this featured is behind feature flag
```

Comment:
fyi, you can add feature flags to cypress here: https://github.com/elastic/kibana/blob/4d353f08769b15cb764e2903bf786b401620b8ed/x-pack/test/security_solution_cypress/config.ts#L49

Reply:
Thanks @michaelolo24 , but since, this feature will not be used by our customers when it is behind the feature, I will prefer other thing to be tested.

Additionaly, Glo has agreed to help with deploying tests with `both` feature flag on and off so that both of the scenarios can be tested. So let's see when that happens.

Thanks for the review